.circleci/config.yml

```
@@ -1,7 +1,79 @@
 version: 2.1
 orbs:
-  node: circleci/node@3.0.0
+  node: circleci/node@1.1.6
+jobs:
+  prod-build:
+    docker:
+      - image: uwbpparamedics/uw-blueprint-paramedics-react:0.0.1
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run: npm install
+      - setup_remote_docker:
+          docker_layer_caching: false
+      - run:
+          name: Install docker-compose
+          command: |
+            set -x
+            curl -L https://github.com/docker/compose/releases/download/1.12.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+            chmod +x /usr/local/bin/docker-compose
+      - run:
+          name: Run Docker compose
+          command: |
+            set -x
+            docker-compose -f docker-compose-prod.yaml up -d
+      - persist_to_workspace:
+          root: .
+          paths:
+            - .
+
+  dev-build:
+    docker:
+      - image: uwbpparamedics/uw-blueprint-paramedics-react:0.0.2
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run: npm install
+      - setup_remote_docker:
+          docker_layer_caching: false
+      - run:
+          name: Install docker-compose
+          command: |
+            set -x
+            curl -L https://github.com/docker/compose/releases/download/1.12.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+            chmod +x /usr/local/bin/docker-compose
+      - run:
+          name: Creating mock env file
+          command: touch paramedics-react.env
+      - run:
+          name: Run Docker compose
+          command: |
+            set -x
+            docker-compose -f docker-compose-dev.yaml up -d
+      - persist_to_workspace:
+          root: .
+          paths:
+            - .
+
+  dev-lint:
+    docker:
+      - image: uwbpparamedics/uw-blueprint-paramedics-react:0.0.2
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Lint
+          command: npm run lint
+
 workflows:
-  node-tests:
+  version: 2
+  build-and-lint:
     jobs:
-      - node/test
+      - prod-build
+      - dev-build
+      - dev-lint:
+          requires:
+            - dev-build
```

ah i removed all the explicit `npm install` calls because i assumed we would want all jobs to run independently (as this should be more performant/faster) - i would at least try that out and compare runtimes; i'm also open to sticking with this or other options if they work out better
```suggestion
      - dev-lint
```

---

I tried this and this does not work. Since the npm packages or the docker image is not cached anywhere, we need to run `npm install` in order to run this in parallel.